src/components/ManagerPage/HookDeviceChange.js

```diff
@@ -0,0 +1,69 @@
+// @flow
+
+import { PureComponent } from 'react'
+import { createStructuredSelector } from 'reselect'
+import { connect } from 'react-redux'
+import { getCurrentDevice } from 'reducers/devices'
+import type { Device } from 'types/common'
+
+const hookDeviceChangeInstances = []
+let frozen = 0
+export class FreezeDeviceChangeEvents extends PureComponent<{}> {
+  componentDidMount() {
+    frozen++
+  }
+  componentWillUnmount() {
+    frozen--
+    if (!frozen) {
+      hookDeviceChangeInstances.forEach(i => i.onUnfreeze())
+    }
+  }
+  render() {
+    return null
+  }
+}
+
+/* eslint-disable react/no-multi-comp */
+class HookDeviceChange extends PureComponent<{
+  device: ?Device,
+  onDeviceDisconnected: () => void,
+  onDeviceChanges: Device => void,
```

Comment:
`onDeviceChange` may be more consistent with other `onChange` etc.

(importance: LOW :p)